lib/vsc/install/commontest.py

```diff
@@ -63,6 +64,11 @@
     except ImportError:
         pass
 
+# Prospector doesn;t have support for 3.5 / 3.6
+# https://github.com/PyCQA/prospector/issues/233
+if sys.version_info >= (3, 5):
+    HAS_PROTECTOR = False
```

Comment:
can you fix the typo everywhere? (@JensTimmerman unless this is intentional)

Reply:
The doesn;t is my mistake, and I'd be happy to fix that!